doc/source/MANUAL.rst

```diff
@@ -103,6 +105,22 @@ specified and pointing to an existing file or the default location
 be ignored. Configuration embedded in a ``tox.ini`` will only be used if other
 configuration files are not present.
 
+pyproject.toml
+''''''''''''''
+
+Similarly, if your project is using ``pyproject.toml``, you may forego the
+config file, and instead create a ``[tool.stestr]`` section with the desired
+configuration options.  For example::
+
+  [tool.stestr]
+  test_path = "./project/tests"
+  top_dir = "./"
+  group_regex = "([^\.]*\.)*"
+
+The same caveats apply as the :ref:`tox` with regards to CLI arguments.  Also
+of note is that files specified with ``--config-file``/``-c`` may be ``.ini``
+or TOML format.
```

Comment:
Personally, I'd probably prefer to restrict the toml support to just the pyproject.toml file and leave the `-c` argument only for `ini` format. But I can see how people might prefer `toml` to `ini` and it's not like this real hurts anything. But we should be a bit more explicit here, and say basically this only works if the file extension is `.toml` and the section has to be the same as the pyproject.toml example above `[tool.stestr]` just to make it clear how the toml format support for arbitrary config files works.

Reply:
That's fair; I'll see what I can come up with.